docs/internal/admin/List-Core-Actions.md

```diff
@@ -0,0 +1,288 @@
+   - Condition: Always
+
+3. Trim duplicated responses
+
+   - Action: "trimResponses"
+   - Description: "**CORE-ACTION** Remove duplicated and unchanged responses or ones without a decision made when review is re-submitted by the reviewer."
+   - Condition: Always
+
+4. Update Review Statuses
+
+   - Action: "updateReviewsStatuses"
+   - Description: "**CORE-ACTION** Update other review status to PENDING or CHANGES REQUESTED after one review is submitted to another reviewer in the chain of review-levels"
+   - Condition: Always
+
+5. Increment Application Stage
+
+   - Action: "incrementStage"
+   - Description: "**CORE-ACTION** Increment stage of application when last level reviewer submits review with decision of CONFORM or NON_CONFORM"
+   - Condition: When Reviewer is_last_level & decision is either CONFORM or NON_CONFORM
+
+6. Generate Review Assignments
+
+   - Action: "generateReviewAssignments"
+   - Description: "**CORE-ACTION** Generate next level (in current stage) of review assignments after one reviewer submits a review to next level - when applicable"
+   - Condition: Always
+
+7. Change Application Status (for applicant - LOQ)
+
+   - Action: "changeStatus"
+   - Description: "**CORE-ACTION** Change application status to CHANGES_REQUIRED once a last level reviewer submits with decision of LIST_OF_QUESTIONS"
+   - Condition: Reviwer is_last_level & decision is LIST_OF_QUESTIONS
+
+8. Change Application Outcome (on last stage submission)
+
+   - Action:
+   - Description: "**CORE-ACTION** Change application outcome when last level and last stage - usually a final decision - submits a review as CONFORM or NON_CONFORM. Outcome will be APPROVED or REJECTED accordingly"
+   - Condition: When Reviewer is_last_stage & is_last_level & decision is either CONFORM or NON_CONFORM
+
+9. Change Application Outcome (on Rejection in early stage)
+
+### Other actions
+
+#### Trigger: ON_APPLICATION_SUBMIT
+
+- Asynchronous console log
+
+1. **TODO** Add example of PDF generation
+
+##### Actions
+
+- Asynchronous console log
+  - Action: "consoleLogger"
+  - Description: "Example: console log with application serial and template code"
+  - Condition: Always
+
+#### Trigger: ON_REVIEW_SUBMIT
+
+1. Change Application Outcome (on Rejection in early stage)
+2. Generate new Organisation when APPROVED
+3. Add User to new Organisation
+4. Grant Permissions to User in Organisation
+5. Send Email to User when APPROVED
+6. Send Email to User when REJECTED
+7. Send Email to User when LIST_OF_QUESTIONS
+8. **TODO** Add example of expiry
+9. **TODO** Add example of certificate
+
+##### Actions
+
+1. Change Application Outcome (on Rejection in early stage)
+
+   - Action: "changeOutcome"
+   - Description: "Example action - Change outcome to REJECT in case of NON_CONFORM decision in first stage of Screening."
+   - Condition: When Reviewer is_last_level & stage is Sreening & decision is NON_CONFORM
```

Comment:
Maybe add a comment that the database automatically updates the Status to "COMPLETED" when Outcome is changed from "PENDING", so it's clear why we're not changing status here.

Reply:
Yes, will do. Thanks